fold-and-r.csl

```diff
@@ -140,6 +140,16 @@
           <text variable="URL" prefix=" "/>
         </else>
       </choose>
+      <choose>
+        <if type="speech">
+          <group prefix=", at "  delimiter=", ">
```

Comment:
Thanks for the tip. I've changed it further, since I realized that it was probably better just to have a separate `else-if` for the speech type.

(Every time I return to editing these files, I have to re-learn whatever little part of the csl I'm playing with.)